README.md

```diff
@@ -206,6 +206,7 @@ POWER TOYS:
                            N=1: URL, N=2: URL and tag, N=3: title,
                            N=4: URL, title and tag
       -j, --json           Json formatted output for -p and search
+      --colors             set output colors (see man page for details)
```

Comment:
Additions to the examples in both the README and man page may be helpful as well.